src/Core/Data/BrewClubDbContext.cs

```diff
@@ -16,7 +17,10 @@ protected override void OnModelCreating(DbModelBuilder modelBuilder)
 			modelBuilder.Entity<Article>().ToTable("Articles");
 			modelBuilder.Entity<Classified>().ToTable("Classifieds");
 			modelBuilder.Entity<Recipe>().ToTable("Recipes");
-			
+
+			modelBuilder.Entity<IdentityUserLogin>().HasKey(u => u.UserId);
+			modelBuilder.Entity<IdentityRole>().HasKey(r => r.Id);
+			modelBuilder.Entity<IdentityUserRole>().HasKey(r => new { r.RoleId, r.UserId });
```

Comment:
These set up the many to many keys, I think.  They were required to avoid a startup exception after I changed the base class.